map_server/CMakeLists.txt

```diff
@@ -94,7 +94,7 @@ if(BUILD_TESTING)
   )
 endif()
 
-ament_export_libraries(map_server_image_loader)
+ament_export_libraries("map_server_image_loader")
```

Comment:
It doesn't have to be in quotes. Since target names can't contain spaces they are usually written without quotes.